.github/workflows/vs2019/build-cbmc.bat

```diff
@@ -16,7 +16,7 @@ echo "Remove strawberry"
 del /s /q "c:/Strawberry/c/bin"
 
 echo "Configure CBMC with cmake"
-cmake -S. -Bbuild -GNinja
+cmake -DCMAKE_BUILD_TYPE=Release -S. -Bbuild -GNinja
```

Comment:
This doesn’t quite work on windows because windows creates a multi config build. Instead you need to pass  `--config Release` to `cmake --build build`